packages/kbn-esql-validation-autocomplete/src/validation/validation.ts

```diff
@@ -754,7 +754,7 @@ function validateCommand(command: ESQLCommand, references: ReferenceMaps): ESQLM
   // do not check the command exists, the grammar is already picking that up
   const commandDef = getCommandDefinition(command.name);
 
-  if (commandDef.validate) {
+  if (commandDef?.validate) {
```

Comment:
ℹ️ Kinda unrelated but it fixes an error in the console. Because the metrics command is still not in the validator it creates an error in the console.